src/components/Input/Input.jsx

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { v4 as uuid } from 'uuid';
+import Cleave from 'cleave.js/react';
+import * as InputMasks from './InputMasks';
+import './Input.scss';
+
+const Input = ({
+  autoComplete,
+  autoFocus,
+  className,
+  id,
+  inputMask,
+  isDisabled,
+  isRequired,
+  label,
+  name,
+  onBlur,
+  onChange,
+  onFocus,
+  placeholder,
+  type,
+  value,
+}) => {
+
+  const handleChange = (e) => {
+    onChange(e);
+  };
+
+  const handleFocus = (e) => {
+    if (onFocus) onFocus(e);
+    e.currentTarget.select();
+  };
+
+  const handleBlur = (e) => {
+    if (onBlur) onBlur(e);
+  };
+
+  const inputClasses = classNames(
+    className,
+    'input',
+    {
+      // [`font-size-${size}`]: size,
+      // [`font-color-${color}`]: color,
+    }
+  );
+
+  const inputId = id || uuid();
+
+  const getInputMask = () => {
+    if (!inputMask) return undefined;
```

Comment:
This function should never fire if `inputMask` is falsey.

Reply:
Yup, I figured we could just use getInputMask consistently everywhere but there is no need to fire it at all so I will just do the conditional check with inputMask